gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/TimingEvent.java

```diff
@@ -89,6 +89,8 @@
     public static final String PROCESSED_COUNT_FIELD = "processedCount";
     public static final String MAX_ATTEMPTS_FIELD = "maxAttempts";
     public static final String CURRENT_ATTEMPTS_FIELD = "currentAttempts";
+    //This state should always move forward
```

Comment:
definitely true... from here want to refer the maintainer to the description in
`KafkaJobStatusMonitor.addJobStatusToStateStore`?